### ISSUE TITLE
Injection of reload script fails when HTML postprocessed

### ISSUE THREAD

**Issue by ahnlak** [Cecilapp/Cecil#1673]
### Describe the bug

If HTML postprocessing is turned on, the output HTML is minified - part of this processing removes the `</body>` and `</html>` tags.

The logic to insert the live reload script when `serve`ing attempts to cope with this, but fails to enclose the script in suitable `<script>` tags, so it doesn't actually work:
https://github.com/Cecilapp/Cecil/blob/6c8a69b13831695afb04a4540d20f0db8eb72db2/resources/server/router.php#L107

(I would just send you a PR, but I haven't delved into building a phar so testing would be ... limited!)

### To reproduce

1. Enable post-processing in `config.yml`
2. Serve with `cecil.phar serve`
3. Open any HTML page, look at the bottom of the page and see 
```
// Cecil's live reload script var evtSource = new EventSource('/watcher'); evtSource.addEventListener('reload', function(e) { let lastChange = e.data; let lastReload = sessionStorage.getItem('livereload'); if (lastChange > lastReload) { sessionStorage.setItem('livereload', Math.floor(Date.now() / 1000)); console.log('reload now!'); location.reload(true); } }, false);
```
4. No live reloading occurs.

### Expected behavior

Code should not be shown, and live reload logic should work as normal.

### Version

7.36.0

**Comment by ahnlak** [comment]
(actually I poked through your github actions and figured out how to build, so have sent https://github.com/Cecilapp/Cecil/pull/1674 as a fix for this to save you some time!)

**Comment by ArnaudLigny** [comment]
Hello @ahnlak, thanks a lot for your contribution 🤗

I have tested and merged your fix, now avalable in the last release ([7.36.1](https://github.com/Cecilapp/Cecil/releases/tag/7.36.1).
